src/components/elements/AccordionSection.svelte

```diff
@@ -0,0 +1,54 @@
+<script>
+    import RedButton from '../buttons/RedButton.svelte';
+    import { mobile, active } from '../stores.js';
+    export let title;
+    export let sequence;
+    export let text;
+    export let href;
+
+    let mobileValue;
+    mobile.subscribe((value) => {mobileValue = value;});
+
+    let activeValue;
+    active.subscribe((value) => {activeValue = value;});
+</script>
+
+{#if activeValue==sequence}
+
+<div class="h-fit sm:h-full grid grid-cols-1">
+    <div class="grid grid-cols-5 p-4 content-center items-center">
+        {#if mobileValue}
+            <div class="col-span-1 text-6xl me-6 text-center font-menobanner text-mcswf-gold py"> {sequence}</div>
+        {/if}
+        <div class="col-span-3 text-left pt-2 tracking-widest sm:text-center font-bold block text-xl md:text-2xl lg:text-4xl"> {title} </div>
```

Comment:
Weird centering, this needs to be `sm:text-left`.

Reply:
Adjusted as requested.